frame/executive/src/lib.rs

```diff
@@ -769,7 +800,7 @@ mod tests {
 				header: Header {
 					parent_hash: [69u8; 32].into(),
 					number: 1,
-					state_root: hex!("2c01e6f33d595793119823478b45b36978a8f65a731b5ae3fdfb6330b4cd4b11").into(),
+					state_root: hex!("6e70de4fa07bac443dc7f8a812c8a0c941aacfa892bb373c5899f7d511d4c25b").into(),
```

Comment:
out of curiosity, why does the state root change?

Reply:
I added on_idle to the block and it returns a weight of 175, so used weight changes